app/assets/javascripts/select2.coffee

```diff
@@ -70,33 +70,55 @@ $(document).ready ->
 
     $longNameElement = $element.parent().siblings().find('.data-center-long-name')
 
-    $multipleItem = $element.closest('.multiple-item')
-    $dataContactType = $element.closest('.data-contact-type')
+    if $element.hasClass('bulk-updates-data-center')
+      # Bulk Updates form for Find & Update Data Center
 
-    if $dataContactType.length > 0
-      # the select is in the data contact form, so use the data contact type we are in
-      $relatedUrl = $dataContactType.find('.related-urls:first')
+      $relatedUrl = $element.parents('fieldset').find('.related-url')
+      $urlElement = $relatedUrl.find('.bulk-updates-url')
+      $urlContentTypeElement = $relatedUrl.find('.bulk-updates-related-url-content-type-select')
+      $urlTypeElement = $relatedUrl.find('.bulk-updates-related-url-type-select')
+
+      if $urlElement.val() != ''
```

Comment:
how much of this is the same as above? extract whatever you can from the two methods

Reply:
Yeah, there's a good bit of duplicate code, I'll try and consolidate.